src/main/java/com/google/cloud/tools/gradle/appengine/appyaml/AppYamlDeployTargetResolver.java

```diff
@@ -46,24 +47,25 @@
           + APPENGINE_CONFIG
           + " is not allowed for app.yaml based projects";
 
-  private final Gcloud gcloud;
+  private final CloudSdkOperations cloudSdkOperations;
 
-  public AppYamlDeployTargetResolver(Gcloud gcloud) {
-    this.gcloud = gcloud;
+  public AppYamlDeployTargetResolver(CloudSdkOperations cloudSdkOperations) {
+    this.cloudSdkOperations = cloudSdkOperations;
   }
 
   /**
    * Process user configuration of "projectId". If not configured or set to APPENGINE_CONFIG (not
    * allowed for app.yaml based projects), show usage. If set to GCLOUD_CONFIG then read from
    * gcloud's global state. If set but not a keyword then just return the set value.
    */
+  @Override
   public String getProject(String configString) {
     if (configString == null
         || configString.trim().isEmpty()
         || configString.equals(APPENGINE_CONFIG)) {
       throw new GradleException(PROJECT_ERROR);
     } else if (configString.equals(GCLOUD_CONFIG)) {
```

Comment:
tiny nit: can remove `else`s

Reply:
oh yeah, weird that I did that, must be losing my mind.